pipeline.py

```diff
@@ -24,13 +39,16 @@ class MorphologicalToken:
     """
 
     def __init__(self, original_word):
-        pass
+        self.original_form = original_word
+        self.normalized_form = ''
+        self.tags_mystem = ''
+        self.tags_pymorphy = ''
 
     def get_cleaned(self):
         """
         Returns lowercased original form of a token
         """
-        pass
+        return self.original_word.lower()
 
     def get_single_tagged(self):
```

Comment:
you should implement both methods for tagged representation to get the desired mark

Reply:
I know, I just haven't completed the work. I will implement both.